VirtualReality/MRML/vtkMRMLVirtualRealityViewNode.cxx

```diff
@@ -75,6 +76,7 @@ void vtkMRMLVirtualRealityViewNode::WriteXML(ostream& of, int nIndent)
   vtkMRMLWriteXMLFloatMacro(motionSensitivity, MotionSensitivity);
   vtkMRMLWriteXMLBooleanMacro(controllerTransformsUpdate, ControllerTransformsUpdate);
   vtkMRMLWriteXMLBooleanMacro(controllerModelsVisible, ControllerModelsVisible);
+  vtkMRMLWriteXMLBooleanMacro(trackHMD, TrackHMD);
```

Comment:
Properties that specify if a certain action is enabled or not, usually named as `...Enabled`. Could we rename this to HMDTrackingEnabled (hmdTrackingEnabled)?